templates/base.html

```diff
@@ -0,0 +1,132 @@
+      content="Hard Hack is a UCSD hardware-focused hackathon."
+    />
+    <meta
+      name="keywords"
+      content="Hard Hack, UCSD, UCSD Hard Hack, Hard Hack San Diego, Hardware Hackathon, Hard Hack 2023, Hard Hack website, Hard Hack instagram, Hard Hack media, Hard Hack at UCSD, Hard Hack at San Diego, Hackathon, Hackathon UCSD"
+    />
+    <meta name="author" content="Hard Hack" />
+    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="./assets/modules/dropdown/dropdown.css" />
+    <link rel="stylesheet" href="./assets/modules/heading/heading.css" />
+    <script src="./assets/libs/jquery.min.js"></script>
+    <script src="index.js"></script>
+
+    <script src="./assets/modules/dropdown/dropdown.js"></script>
+    <script src="./assets/modules/heading/heading.js"></script>
+
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="./assets/favicon/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="./assets/favicon/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="./assets/favicon/favicon-16x16.png"
+    />
+    <link rel="manifest" href="./assets/favicon/site.webmanifest" />
+    <link
+      rel="mask-icon"
+      href="./assets/favicon/safari-pinned-tab.svg"
+      color="#5bbad5"
+    />
+    <meta name="msapplication-TileColor" content="#da532c" />
+    <meta name="theme-color" content="#ffffff" />
+  </head>
+
+  <!-- START OF BODY -->
+  <body>
+    <div id="menu-bar" class="background-element">
+      <a href="/#about-page"><span>About</span></a>
+      <a href="/#faq-page"><span>FAQ</span></a>
+      <a href="/sponsors"><span>Sponsors</span></a>
+    </div>
+    <div id="dropdown-menu-bar" class="background-element">
+      <div
+        class="hamburger-icon"
+        style="position: absolute; z-index: 1; right: 0px; margin: 1em 1.5em"
+      >
+        <div class="hamburger-bar1"></div>
+        <div class="hamburger-bar2"></div>
+        <div class="hamburger-bar3"></div>
+      </div>
+      <div
+        id="dropdown-menu-bar-items"
+        style="
+          background-color: #158dc1;
+          position: relative;
+          width: 100%;
+          height: 0px;
+          overflow: hidden;
+        "
+      >
+        <div>
+          <div style="height: 68px"></div>
+          <a href="/#about-page"><p>About</p></a>
+          <a href="/#faq-page"><p>FAQ</p></a>
+          <a href="/sponsors"><p>Sponsors</p></a>
+        </div>
+      </div>
+    </div>
+    <div id="background">
+      <img
+        class="background-element"
+        src="./assets/images/hard_hack_small_logo.png"
+        style="width: 5%; min-width: 50px; top: max(3vh, 20px); left: 20px"
+      />
+    </div>
+    {% block content %}
+    <p>Put something here!</p>
+    {% endblock content %}
+    <!-- SOCIALS -->

```

Comment:
Maybe we think of something else to put here.
We might put Andrew's questions box instead?